src/service/promql/engine.rs

```diff
@@ -250,15 +250,21 @@ impl QueryEngine {
             None => return Ok(vec![]),
         };
 
+        let end = self.start + (self.interval * self.time_window_idx); // 15s
+        let start = end - self.lookback_delta; // 5m
```

Comment:
[nit, optional]
```suggestion
        let start = end - self.lookback_delta; // e.g., 5m
```

Rationale: it's not always `5m` and `15s`, AFAIU.

Reply:
just delete the comment